portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,5 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+/* https://www.w3schools.com/w3css/tryit.asp?filename=tryw3css_templates_cv&stacked=h */
```

Comment:
Could you move this to the top and add a more detailed comment here?

Reply:
I have addressed this issue in the "Move reference to top and add more comments"